src/components/TopBar.js

```diff
@@ -14,10 +14,11 @@ import { Component } from 'src/libs/wrapped-components'
 
 const styles = {
   topBar: {
-    flex: 'none', height: 80,
+    flex: 'none', height: 100,
     backgroundColor: 'white', paddingLeft: '1rem',
     display: 'flex', alignItems: 'center',
-    borderBottom: `2px solid ${colors.blue[0]}`
+    borderBottom: `2px solid ${colors.blue[0]}`,
+    boxShadow: Style.standardShadow, zIndex: 1
```

Comment:
Can we do this without the zIndex? They're tough to keep track of once they're introduced.